web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1841,6 +1841,17 @@
                   : "";
             return angular.isFunction(this[fnName]) ? this[fnName]() : false;
           },
+          canViewMetadataHistory: function () {
+            var profile = gnConfig["metadata.history.accesslevel"] || "Editor",
+              fnName =
+                profile !== ""
+                  ? "is" + profile[0].toUpperCase() + profile.substring(1) + "OrMore"
+                  : "";
+            if (profile ==="RegisteredUser") {
+              return true;
+            }
+            return angular.isFunction(this[fnName]) ? this[fnName]() : false;
+          },
```

Comment:
Please update the file with prettier formatting.

Reply:
The format was indeed lining with all methods. It was copy pasted. Here is what my IDE looks like. The pretty formatting will need to be at the whole file level not limited to the code that I commit.

![image](https://github.com/geonetwork/core-geonetwork/assets/74916635/ce505f37-b835-4fd8-84b1-5049e008c01d)